example-monorepo/packages/plugin-test/.env.schema

```diff
@@ -0,0 +1,27 @@
+# @defaultRequired=infer @defaultSensitive=false
+# @generateTypes(lang=ts, path=env.d.ts)
+#
+# @currentEnv=$APP_ENV
+# # @envFlag=APP_ENV
+# # @badDecorator
+#
+# @import(./.env.op)
+# ---
+
+# our environment flag
+# @type=enum(dev, preview, prod, test)
+# @docsUrl=https://example.com/docs/app_env
+APP_ENV=dev
+
+# some description
+# @required=true
+# @docs(https://example1.com)
```

Comment:
`@docsUrl` is now deprecated in favor of a `@docs()` function, which can be called multiple times.

Reply:
makes sense to me. I think since there are multiple deprecations in this release we should probably consider a proper minor (0.1.0?) release

Reply:
So far my thinking has been that we bump up to 0.1.0 (or maybe 1.0.0?) when we reach a point where we are ready to guarantee stability, and otherwise just keep rolling with it. But open to other ideas. Should also probably write something about our versioning strategy somewhere in the docs?